src/test_l1_rmfAudioCapture.c

```diff
@@ -1799,37 +1420,31 @@ static UT_test_suite_t * pSuite = NULL;
  */
 int test_l1_rmfAudioCapture_register_positive_only_suite ( void )
```

Comment:
we can add easy way to add suite 
so remove this api also

Reply:
done